packages/googlesheets/src/Adaptor.js

```diff
@@ -233,6 +233,30 @@ export function getValues(spreadsheetId, range, callback = s => s) {
   };
 }
 
+
+/**
+ * Exposes the googlesheets client for more flexible job writing
```

Comment:
I hadn't realised our docs for `fn()` were so bad :see_no_evil: 

When we're done I'll do a pass on these docs (then raise an issue to fix it in common)